release-1.8/release_notes_draft.md

```diff
@@ -211,19 +229,15 @@ kind.
 
 #### Cluster Discovery/Bootstrap
 
-* [beta] The authentication and verification mechanism called Bootstrap Tokens has been added to the core API, which can be used to easily add new members to a cluster. ([#130](https://github.com/kubernetes/features/issues/130), [@luxas](https://github.com/luxas), [@jbeda](https://github.com/jbeda)).
-
-#### Addons
-* [alpha] A new system addon manager is available that is aiming to improve the downsides of the existing `kube-addons.sh` manager. ([#18](https://github.com/kubernetes/features/issues/18), [@justinsb](https://github.com/justinsb))
+* [beta] The authentication and verification mechanism called Bootstrap Tokens has been added to the core API, which can be used to easily add new node identities to a cluster. ([#130](https://github.com/kubernetes/features/issues/130), [@luxas](https://github.com/luxas), [@jbeda](https://github.com/jbeda)).
 
 #### Multi-platform
-* [beta] Kubernetes now has automated continuous-integration tests against all of our supported platforms (amd64, armhfp, aarch64, ppc64le), to ensure that it continues to work on these platforms. It's also possible to run clusters with nodes of mixed architectures. ([#288](https://github.com/kubernetes/features/issues/288), [@luxas](https://github.com/luxas), [@mkumatag](https://github.com/mkumatag), [@ixdy](https://github.com/ixdy))
+
+* [alpha] The Conformance e2e test suite now passes on the arm, arm64 and ppc64le platforms as well ([#288](https://github.com/kubernetes/features/issues/288), [@luxas](https://github.com/luxas), [@mkumatag](https://github.com/mkumatag), [@ixdy](https://github.com/ixdy))
 
 #### Cloud Providers
-* [beta] Support for out-of-tree and out-of-process cloud providers, a.k.a pluggable providers, has been promoted from alpha to beta. ([#88](https://github.com/kubernetes/features/issues/88), [@wlan0](https://github.com/wlan0))
 
-#### DaemonSet
-* [beta] DaemonSet upgrades can be achieved via a start-then-kill update strategy. ([#373](https://github.com/kubernetes/features/issues/373), [@aaronlevy](https://github.com/aaronlevy), [@diegs](https://github.com/diegs))
+* [alpha] Support for out-of-tree and out-of-process cloud providers, a.k.a pluggable providers, has been significantly improved. Creating an out-of-code cloud provider is now easier and includes more features. ([#88](https://github.com/kubernetes/features/issues/88), [@wlan0](https://github.com/wlan0))
```

Comment:
Should this call out to the limitations of current implementations? For example, is https://github.com/kubernetes/kubeadm/issues/408 solved in 1.8 ? 

This description makes it sound like it is ready.

Reply:
That issue will actually make it in v1.8 I think :smile:
Any suggestion how to reword?

Reply:
> That issue will actually make it in v1.8 I think . Great!

> Any suggestion how to reword?
Something like "For future todos, check here (link to github issue). Essentially, some way to get a sense what is yet not possible and where are those worked on. For example, I read that Volume support is not possible by external CCM in one of the threads about DO CCM.